sdk/cosmos/tests/attachment_00.rs

```diff
@@ -107,26 +107,35 @@ async fn attachment() -> Result<(), azure_cosmos::Error> {
 
     // create reference attachment
```

Comment:
Can this test be moved to using the mock testing framework?

Reply:
Yes, that's the plan - but per https://github.com/Azure/azure-sdk-for-rust/pull/375#issuecomment-945826374 we don't yet have a cosmosdb that exposes this API against which we can run the tests.